### PR TITLE
Remove SASS dependency; Bundle CSS with esbuild.

### PR DESCRIPTION
**Pull request by dbauszus-glx**
Color variables are defined on the `:root{}`

CSS file imports must use `url()`

SVG must use a loader `--loader:.svg=dataurl`

Nesting requires an ampersand.

CSS can be bundled with:

```
npx esbuild --bundle public/css/_mapp.css --outfile=public/css/mapp.css
```

and

```
npx esbuild --bundle public/css/_ui.css --outfile=public/css/ui.css --loader:.svg=dataurl
```

### CODE REVIEW

**Comment by RobAndrewHurst** [comment]
I noticed that the colour of controls are lighter than before!

image

image

Are we happy with this?

@simon-leech @dbauszus-glx @cityremade @eo-uk @jfitzpatrick17

**Review by simon-leech — CHANGES_REQUESTED** [review]
I think that the lighter grey on the mask-icons suggests they are inactive. What about using the primary colour blue for the icons? `--color-primary: #003D57`
image

**Comment by RobAndrewHurst** [comment]
@dbauszus-glx @simon-leech I fixed the issue where we were seeing masked-icons lighter than they should be. It was just incorrect css.

image

image

**Review by RobAndrewHurst — APPROVED** [review]
## Code Review for New CSS Changes
😃  I've just wrapped up a thorough review of the recent implementation of our advanced CSS practices Here's my feedback:

Modular Color Management 🎨:
The decision to define color variables using the :root{} selector is very nice! 🎨 This practice guarantees consistent color usage across all our projects, making it simpler for our awesome developers to manage and update color palettes. Fantastic job! 🎉

Precise Resource Importing 💎:
Switching to the url() function for CSS file imports is also really good. 💼 This change not only enhances resource loading efficiency but also elevates the overall readability of our codebase.🧹

Enhanced Nesting 🐣:
Thumbs up for promoting the use of the ampersand & notation for nesting! 🏗️ This practice not only tidies up our stylesheets but also supports better maintenance down the road. Let's keep the ball rolling in terms of this structure style! 📐

Efficient Bundling with esbuild 📦:
The integration of the esbuild bundler for CSS optimization is a wise move. 🚀 

In Conclusion
I'm very stoked with the direction we're taking with these CSS practices. Going forward this will help us producce top-tier, efficient, and maintainable code. Lekker stuff 🎉🚀

This will all get merged in when @cityremade & @simon-leech Have all given their review! 😃

**Review by simon-leech — APPROVED** [review]

**Comment by simon-leech** [comment]
All good with me!
